cloud

```diff
@@ -702,23 +717,27 @@ function scheduler_job_withdraw() {
 }
 
 # Deploy to scheduler
-# Args: project prefix trigger_topic
+# Args: project prefix load_queue_trigger_topic pick_notifications_trigger_topic
 function scheduler_deploy() {
     declare -r project="$1"; shift
     declare -r prefix="$1"; shift
-    declare -r trigger_topic="$1"; shift
+    declare -r load_queue_trigger_topic="$1"; shift
+    declare -r pick_notifications_trigger_topic="$1"; shift
     # Deploy the jobs
-    scheduler_job_pubsub_deploy "$project" "${prefix}trigger" \
-                                "$trigger_topic" '* * * * *' '{}'
+    scheduler_job_pubsub_deploy "$project" "${prefix}load_queue_trigger" \
+                                "$load_queue_trigger_topic" '* * * * *' '{}'
+    scheduler_job_pubsub_deploy "$project" "${prefix}pick_notifications_trigger" \
+                            "$pick_notifications_trigger_topic" '30 * * * *' '{}'
```

Comment:
Could you please indent this line the same way the previous ones are indented, and wrap the arguments so that the lines are 79 characters max?